src/Components/ArticlesList.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import getAllArticles from "../api";
+
+const ArticlesList = () => {
+    const [articlesList, setArticlesList] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
+
+    useEffect(() => {
+        getAllArticles().then((allArticles) => {
+            setArticlesList(allArticles)
+            setIsLoading(false)
+        })
+    }, [])
+
+    if (isLoading) {
+        return <h2>Loading...</h2>
+    }
+    else {
+        return (
+          <main>
+                <h2>Articles</h2>
+            <ul>
+              {articlesList.map((article) => {
+                  return (
+                    <li key={article.article_id} className="articles">
+                      <h3>{article.title}</h3>
+                      <img src={article.article_img_url} alt="article image" />
+                      <p>{article.author}</p>
+                      <p>{article.topic}</p>
```

Comment:
Also your css is decent so far but I recommend making it a bit more obvious what the information is. You could use icons or different fonts to indicate which of these is the author and which is the topic
Also later on we will want to sort these by things like comment count, date created and number of likes/votes, so these things are worth adding to your article cards at some point